### DISCUSSION TITLE
How to show more on the output?

### DISCUSSION THREAD

**Discussion by GF-Huang** [dotnet/interactive#2594] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/4510984/211161967-ec4b08fd-2ad6-4085-8a8b-6b438730af6a.png)

**Comment by jonsequitur** [comment, marked as answer]
You can change this behavior by setting `Microsoft.DotNet.Interactive.Formatting.Formatter.ListExpansionLimit`.

More details here: https://github.com/dotnet/interactive/blob/main/docs/formatting.md
